src/pages/[platform]/build-a-backend/functions/custom-functions/index.mdx

```diff
@@ -0,0 +1,220 @@
+
+export const meta = {
+  title: 'Custom functions',
+  description:
+    'Use another AWS Lambda runtimes like python, golang to perform tasks and customize workflows.',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ]
+};
+
+export function getStaticPaths() {
+  return getCustomStaticPath(meta.platforms);
+}
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+AWS Amplify Gen 2 Functions are AWS Lambda functions that can be used to perform tasks and customize workflows in your Amplify app.Functions can be written in Node.js, Python, Go, or any other language supported by AWS Lambda:
+
+<Callout warning>
+
+**Note:** Custom runtimes are not supported in Amplify Functions directly. If you need docker support to use a custom runtime for example in Python, Docker is not supported in Amplify Hosting and Amplify backend auto build. You shouldn't use docker build in your function.There is an example of how to use python in a lambda function in the without Docker section below.
+
+</Callout>
+
+Technically, you can use any language supported by AWS Lambda. These are NodeJS, Python, Java, .NET, Ruby.
+To use other languages in Lambda, such as Go or Rust, use an OS-only runtime.
+
+In this guide, you will learn how to create python and golang functions in Amplify Functions.
+
+# Python functions
+To get started, create a new directory and a resource file, `amplify/functions/say-hello/resource.ts`. Then, define the Function with `defineFunction`:
+
+```ts title="amplify/functions/say-hello/resource.ts"
+import { defineFunction } from "@aws-amplify/backend";
+import { DockerImage, Duration } from "aws-cdk-lib";
+import { Code, Function, Runtime } from "aws-cdk-lib/aws-lambda";
+import { execSync } from "child_process";
+import * as path from "path";
+
+export const sayHello = defineFunction(
+  (scope) =>
+    new Function(scope, "say-hello", {
+      handler: "index.handler",
+      runtime: Runtime.PYTHON_3_9, // or any other python version
+      timeout: Duration.seconds(20), //  default is 3 seconds
+      code: Code.fromAsset(functionDir, {
+        bundling: {
+          image: DockerImage.fromRegistry("dummy"),
+          local: {
+            tryBundle(outputDir: string) {
+              execSync(
+                `python3 -m pip install -r ${path.join(functionDir, "requirements.txt")} -t ${path.join(outputDir)} --platform manylinux2014_x86_64 --only-binary=:all:`
+              );
+              execSync(`rsync -rLv ${functionDir}/* ${path.join(outputDir)}`);
+              return true;
+            },
+          },
+        },
+      }),
+    })
+);
+```
+
+Next, create the corresponding handler file at `amplify/functions/say-hello/handler.ts`. This is where your function code will go.
```

Comment:
Shouldn't this be `amplify/functions/say-hello/index.py` instead of `amplify/functions/say-hello/handler.ts`?